package.json

```diff
@@ -1,71 +1,54 @@
 {
-  "name": "microsoft-graph-explorer",
-  "version": "1.5.0",
-  "description": "Microsoft Graph Explorer",
+  "name": "g-e",
+  "version": "0.0.0",
   "scripts": {
-    "build": "tsc -p src/",
-    "build:watch": "npm run build -- -w",
-    "build:aot": "ngc -p src/tsconfig-aot.json",
-    "build:prod": "npm run build:aot && npm run rollup",
-    "start": "concurrently \"npm run build:watch\" \"npm run serve\"",
-    "serve": "lite-server -c=bs-config.json",
-    "test": "npm run build && karma start karma.conf.js --single-run --browsers ChromeHeadless",
-    "test:watch": "concurrently \"npm run build:watch\" \"karma start karma.conf.js\"",
+    "ng": "ng",
+    "start": "ng serve -o",
+    "build": "ng build",
+    "test": "ng test --watch=false --browsers=ChromeHeadless",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
```

Comment:
What happened to the build:aot vis build:prod?  Are those not needed any more? What were they used for?